README.md

```diff
@@ -64,28 +64,39 @@ tools:
 
 * Mailing lists:
 
-  * **janusgraph-users (at) googlegroups.com**
-    ([archives](https://groups.google.com/group/janusgraph-users))
+  * **janusgraph-users (at) lists.lfaidata.foundation**
+    ([archives](https://lists.lfaidata.foundation/g/janusgraph-users/topics))
     for questions about using JanusGraph, installation, configuration, integrations
 
-    To join with a Google account, use the [web
-    UI](https://groups.google.com/forum/#!forum/janusgraph-users/join); to
+    To join with a LF AI & Data account, use the [web
+    UI](https://lists.lfaidata.foundation/g/janusgraph-users/join); to
     subscribe/unsubscribe with an arbitrary email address, send an email to:
 
-    * janusgraph-users+subscribe (at) googlegroups.com
-    * janusgraph-users+unsubscribe (at) googlegroups.com
+    * janusgraph-users+subscribe (at) lists.lfaidata.foundation
+    * janusgraph-users+unsubscribe (at) lists.lfaidata.foundation
 
-  * **janusgraph-dev (at) googlegroups.com**
-    ([archives](https://groups.google.com/group/janusgraph-dev))
+  * **janusgraph-dev (at) lists.lfaidata.foundation**
+    ([archives](https://groups.google.com/group/janusgraph-dev/topics))
```

Comment:
This is pointing to Google Groups instead of LF AI & Data; please update the URL.